framework/Models/Code.py

```diff
@@ -692,32 +713,26 @@ def collectOutputFromDict(self,exportDict,output,options=None):
       @ Out, None
     """
     prefix = exportDict.pop('prefix')
-    #convert to *spaceParams instead of inputs,outputs
-    if 'inputs' in exportDict.keys():
-      inp = exportDict.pop('inputs')
-      exportDict['inputSpaceParams'] = inp
-    if 'outputs' in exportDict.keys():
-      out = exportDict.pop('outputs')
-      exportDict['outputSpaceParams'] = out
-    if output.type == 'HDF5':
-      output.addGroupDataObjects({'group':self.name+str(prefix)},exportDict,False)
+    if 'inputSpaceParams' in exportDict.keys():
+      inKey = 'inputSpaceParams'
+      outKey = 'outputSpaceParams'
```

Comment:
why do we need this?

Reply:
Seems to me, we do not need this method. This is just a conversion of this method to use the new data objects. Personally, I think we need to remove this method.

Reply:
can you remove it?

Reply:
Actually I can remove it when I address the HDF5 thing